src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -227,8 +227,8 @@
             </div>
 
             <!-- orgPerson we haven't touched: -->
-            <div v-else class="actions">
-              <span class="edit-action" :class="{'pr-4': hasRoleProprietor(orgPerson)}">
+            <div v-else class="actions mr-4">
+              <span class="edit-action" :class="{'pr-4': canRemove(orgPerson)}">
```

Comment:
This is to support alignment with and without the more actions (drop-down) menu.